testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1788,63 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', ()=> {
+    QUnit.test('selectByClick option should be changed during initialization', function(assert) {
```

Comment:
This test doesn't make sence. I think it should be contain the test scenario as in the next tests.

Reply:
Removed this test